### DISCUSSION TITLE
Wireless Signal Analog Domain Representations - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#628] [Showcases]
Discuss the "Wireless Signal Analog Domain Representations" showcase here. Ideas and recommendations are welcome.

**Comment by pietrotedeschi** [comment]
@adamgeorge309 when I test the following use case scenario AnalogModelShowcaseNoiseNetwork, I receive the following error:
<!> Unknown parameter 'gainFunctionCacheLimit' -- in module (inet::physicallayer::NoiseDimensionalTransmitter) AnalogModelShowcaseNoiseNetwork.noiseSource.transmitter (id=281), during network initialization

Omnet version: 5.6.2
OS: Ubuntu 19.10
